src/main/java/com/CezaryZal/api/body/manager/repo/BodySizeRepoService.java

```diff
@@ -19,15 +20,19 @@ public BodySizeRepoService(BodySizeRepository BSRepository) {
         this.bodySizeR = BSRepository;
     }
 
-    public BodySize getBodyById(Long id) {
+    protected BodySize getBodyById(Long id) {
         return bodySizeR.findById(id)
                 .orElseThrow(() -> new BodySizeNotFoundException("Body size not found by id"));
     }
 
     public LocalDate getDateLastMeasureByUserId(Long userId) {
-        Date tmpDate = bodySizeR.findDateLastMeasureByUserId(userId)
-                .orElseThrow(() -> new BodySizeNotFoundException("Body size not found by user id"));
-        return tmpDate.toLocalDate();
```

Comment:
Czemu sięz tego wycofałeś? To wygląda lepiej niż aktualna wersja. Jaki efekt chcesz wywołać jak zwrócony będzie null? Chyba powinien właśnie polecieć exception.

Reply:
Ogółem tak, ale ta konkretna metoda zwraca ostatni wykonany pomiar.
Potrzebuje go zawsze przy wyświetleniu informacji o dniu.
A zdarzy się sytuacja że użtkownika założył konto i nie ma jeszcze żadnego pomiaru ale chciałby już obsługiwać dnie.
W takim wypadku wyrzuca błąd bo nie może znaleść recordu, a to nie jest błąd.
Dalej to rozwiązałem.